test/e2e-setup/bicep/modules/cluster.bicep

```diff
@@ -95,6 +98,7 @@ resource hcp 'Microsoft.RedHatOpenShift/hcpOpenShiftClusters@2024-06-10-preview'
     }
     api: {
       visibility: 'Public'
+      authorizedCidrs: authorizedCidrs
```

Comment:
What would an empty list as a value of `authorizedCidrs` mean? Would it basically block all traffic to the API, disable the feature or report an error?

I'm asking because I'm little confused, as this almost looks as if an empty list here disables the feature (so that access is possible from all IP addresses), which is not what would I expect.